packages/main/src/themes/sap_horizon/Panel-parameters.css

```diff
@@ -10,4 +10,5 @@
 	--_ui5_panel_icon_color: var(--sapButton_Lite_TextColor);
 	--_ui5_panel_focus_offset: -1px;
 	--_ui5_panel_content_padding: 0.625rem 1rem;
+	--_ui5_panel_header_background_color: var(--sapToolbar_Background);
```

Comment:
Here, isn't it --sapGroup_TitleBackground if I have understood you right which one should be used for the header.